src/client/components/CreateBountyForm.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+import contracts from 'eth/contracts'
+
+export default inject('CreateBountyFormStore')(observer(props => {
+
+  const { CreateBountyFormStore } = props
+  const { title, description } = CreateBountyFormStore
+
+  const onTitleChangeHandler = (evt) => {
+    CreateBountyFormStore.updateTitle(evt.target.value)
+  }
+
+  const onDescriptionChangeHandler = (evt) => {
+    CreateBountyFormStore.updateDescription(evt.target.value)
+  }
+
+  const onClickHandler = async () => {
+    const { BugBounty, BugBountyFactory } = contracts()
+    const bugBountyFactory = await BugBountyFactory.deployed()
+
+    const createTx = await bugBountyFactory.createBugBounty(
+      500,
+      400,
+      300,
```

Comment:
What are these numbers?

Reply:
@jpkcambridge placeholders for payout amounts for high/critical/medium, etc